### ISSUE TITLE
Fix S4061: Rule should not report when method is interop

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#1283]
### Description

The following declaration should not raise any issue:
```
[DllImport("msvcrt40.dll")]
public static extern int printf(string format, __arglist);
```
